### PR TITLE
ELEMENTS-517: fix styles of user info in user management

### PR DESCRIPTION
**Pull request by afjusto**

### CODE REVIEW

**Comment by Gabez0r** [comment]
👎

**Comment by guirenard** [comment]
We can't include `nuxeo-styles` in nuxeo-elements

File: nuxeo-user-group-management/nuxeo-view-user.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
 -->
 <dom-module id="nuxeo-view-user">
   <template>
-    <style>
+    <style include="iron-flex nuxeo-styles">
```
Comment (adilioaraujo) [inline nuxeo-user-group-management/nuxeo-view-user.html]: At some point we need to remove nuxeo-styles from UI elements

**Comment by adilioaraujo** [comment]
:+1:

**Comment by afjusto** [comment]
¯\_(ツ)_/¯ it's already there

**Comment by Gabez0r** [comment]
👍

**Comment by afjusto** [comment]
https://jira.nuxeo.com/browse/ELEMENTS-494

File: nuxeo-user-group-management/nuxeo-view-user.html (hunk 1 above)
Comment (afjusto) [reply nuxeo-user-group-management/nuxeo-view-user.html]: https://jira.nuxeo.com/browse/ELEMENTS-494

**Pull request merged**